modules/openxr/openxr_interface.cpp

```diff
@@ -46,6 +51,31 @@ uint32_t OpenXRInterface::get_capabilities() const {
 	return XRInterface::XR_VR + XRInterface::XR_STEREO;
 };
 
+PackedStringArray OpenXRInterface::get_suggested_tracker_names() const {
+	PackedStringArray arr;
+
+	// These are hardcoded in OpenXR, note that they will only be available if added to our action map
+
+	arr.push_back("left_hand"); // /user/hand/left is mapped to our defaults
+	arr.push_back("right_hand"); // /user/hand/right is mapped to our defaults
+	arr.push_back("/user/treadmill");
```

Comment:
I think you can use a list initialization for PackedStringArray now:
```
PackedStringArray arr = {
	"left_hand",
	"right_hand",
	"/user/treadmill",
...
};
```

Reply:
Done, also removed the other entries, they should not be there until we have the HTC tracker stuff sorted